app/routes/user/email_circulars.server.ts

```diff
@@ -0,0 +1,62 @@
+import { tables } from '@architect/functions'
+
+export async function createGcnCircularEmailNotification(
```

Comment:
Remove `Gcn` from all method names.

```suggestion
export async function createCircularEmailNotification(
```